src/struct.ts

```diff
@@ -116,8 +116,11 @@ export function parseKey (str: string, ptr: number, end = '='): [ string[], numb
 	return [ parsed, skipVoid(str, endPtr + 1, true, true) ]
 }
 
-export function parseInlineTable (str: string, ptr: number, depth: number = -1): [ Record<string, TomlPrimitive>, number ] {
-	let res: Record<string, TomlPrimitive> = {}
+export function parseInlineTable(
+	str: string, ptr: number,
+	{ depth = -1, integerParsing }: { depth?: number, integerParsing?: IntegerParsing|undefined } = {}
```

Comment:
!! objects as arguments are nice in some places, but very bad here as it prevents minifiers from making everything a single-letter variable. Getting rid of the -1 default would be ideal :/

Reply:
Except for exceptionally stable interfaces (such as string.substring for instance), I think all optional parameters should be in keyword option parameters.
But even short of that, I think positional-only parameters are bad as soon as two independent parameters are optional - as was visible from the use of `void 0` somewhere (maybe in tests) to skip a parameter.
We can make depth required and integerParsing the only optional, but the same problem would bite us back as soon as the next option is implemented...

Reply:
Those functions are internal and won't ever be exposed to public-facing APIs, and the code is small enough that dealing with interfaces changing is a non-issue. The code size impact does matter a lot to me. It must be *smol*.

All parameters can be marked as required, again since it's internal and the fact they're optional is only something the user sees in the public APIs.

Reply:
extractValue's `end` parameter is now required with undefined being allowed, which is slightly regrettable but better than putting it away from ptr.